### ISSUE TITLE
Request: inline comments for overriding key codes

### ISSUE THREAD

**Issue by modess** [codethread/qmk.nvim#14]
Instead of doing this in the nvim lua files:

```
{
	comment_preview = {
		keymap_overrides = {
			['LSG%(KC_GRAVE%)'] = 'Next Window',
		},
}
```

It would be really nice if it could be done directly in the keymap file, something like this perhaps:

```
// qmk.nvim: LSG(KC_GRAVE) | Next Window
```

It would be easier to maintain and also be reproducable.

**Comment by codethread** [comment]
nice idea! something like [this](https://github.com/jurgen-kluft/go-qmk-keymap#configuration) ?

I'd still allow overrides in config, but in-file keymaps would take precedence.

Does that sound like what you're after 🙂

**Comment by modess** [comment]
Yes exactly that! :+1:

**Comment by codethread** [comment]
hey @modess 😄 sorry for the long delay on this... 

This should work now, check out the docs https://github.com/codethread/qmk.nvim#examples for `Using inline JSON`.

tl;dr, something like the following at the bottom of you layout allows inline changes

hope it works how you'd hoped! any issues, please reopen, thanks 🚀 

```
/*
qmk:json:start
{
  "layout": [
    "x x x _",
    "x _ ^xx"
  ],
  "comment_preview": {
    "keymap_overrides": {
      "BAR": "BUTTER",
      "FOO": "😎"
    }
  }
}
qmk:json:end
*/
```

**Comment by modess** [comment]
Great job @codethread! Thank you
